pkg/dia/Messages.go

```diff
@@ -43,11 +51,13 @@ type Pairs []Pair
 type Trade struct {
 	Symbol            string
 	Pair              string
```

Comment:
We might remove the redundant fields Trade.Symbol and Trade.Pair at some point, but let's keep them for now.